src/main/java/uk/gov/hmcts/sptribs/common/CommonConstants.java

```diff
@@ -13,7 +13,7 @@ public final class CommonConstants {
     public static final String CIC_CASE_REPRESENTATIVE_NAME = "CicCaseRepresentativeFullName";
     public static final String CIC_CASE_RESPONDENT_NAME = "CicCaseRespondentFullName";
     public static final String CIC_CASE_HEARING_TYPE = "HearingType";
-    public static final String CIC_CASE_HEARING_VENUE = "HearingVenue";
+    public static final String CIC_CASE_HEARING_VENUE = "FlagType";
```

Comment:
Why hearing venue variable hold flag type value? The variable needs to be renamed if we expect to use flag type.

Reply:
Reverted the change